Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -395,7 +395,7 @@ void NetworkProcess::createNetworkConnectionToWebProcess(ProcessIdentifier ident
     for (auto pageID : parameters.pagesWithRelaxedThirdPartyCookieBlocking)
         m_pagesWithRelaxedThirdPartyCookieBlocking.add(pageID);
 
-    if (auto* session = networkSession(sessionID)) {
+    if (auto session = checkedNetworkSession(sessionID)) {
```

Comment:
We can just use `CheckedPtr` and `networkSession()` here and many other places. Not even sure we need a `checkedNetworkSession` given that it may return null and needs to be null checked.

Reply:
Sure, moving to https://github.com/WebKit/WebKit/pull/46203.